### ISSUE TITLE
Rom limit to 15?

### ISSUE THREAD

**Issue by fsegouin** [shilga/rp2040-gameboy-cartridge-firmware#5]
After uploading 15 ROMs, even if there is enough banks free according to the UI, I get the "Uploading the ROM failed" message every time. If I delete 1 ROM I can then add another one. There seems to be a hard limit at 15 ROMs, although I did not see this being mentioned anywhere. Is this normal behavior?

Here's the console dump of the error: `requestRomUploadCommand rejected with code 1`

Thanks!

**Comment by shilga** [comment]
The ROM limit was supposed to be 16. The number is kinda artificial as there is no technical reason for this. The cartridge could support more but currently it's not possible to scroll in the bootmenu, so the limitation is the screen size. But it's something that can be solved in the future.

I did not work on this as in the way I used the cartridge I always ended up on the limit of banks than on the limit of ROMs, so I worked on more important things. But if small ROMs are used potentially a lot of ROMs could be loaded on the cartridge. Tetris for example would fit 444 times.
